CLI/Repl.cpp

```diff
@@ -220,14 +216,18 @@ extern "C"
         // setup state
         setupState(L);
 
+        // sandbox thread
+        luaL_sandboxthread(L);
+
         // run code + collect error
         std::string error = runCode(L, source);
 
         // output error(s)
         if (error.length())
         {
-            fprintf(stdout, "%s\n", error.c_str());
+            return std::move(error.c_str());
```

Comment:
Heads up: this is UB. `error.c_str()` lives for as long as the stack allocated `error` and doesn't return a copy.

Reply:
The alternative to doing this is returning `std::string` but that requires using Embind or WebIDL Binder.

Reply:
Since Emscripten is singlethreaded you can cache the return value via a static, e.g. `static std::string result = error; return result.c_str();`. std::move is superfluous.